### PR TITLE
Apply manual codes

### PR DESCRIPTION
**Pull request by ritazagoni**

### CODE REVIEW

File: code_schemes/Change.json [hunk 1]
```diff
@@ -0,0 +1,78 @@
+{
+  "Codes": [
+    {
+      "CodeID": "code-baec502746d5",
+      "CodeType": "Normal",
+      "DisplayText": "Test",
```
Comment (as2388) [inline code_schemes/Change.json:6]: If you're going to include a test code, then take care to think through the implications first. If you delete this code from the scheme but still have data with this label, then I suspect Coda will give you scary warnings and the pipeline will crash with KeyErrors.

File: run_scripts/test_contact_rapid_pro_ids.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (as2388) [inline run_scripts/test_contact_rapid_pro_ids.json]: Shouldn't this be a list?

File: src/lib/pipeline_configuration.py [hunk 3]
```diff
@@ -11,7 +11,9 @@ def _open_scheme(filename):
         return Scheme.from_firebase_map(firebase_map)
 
 class CodeSchemes(object):
-    pass
+    VALUEABLE = _open_scheme("Valuable.json")
```
Comment (as2388) [inline src/lib/pipeline_configuration.py]:
```suggestion
    VALUABLE = _open_scheme("Valuable.json")
```

File: src/lib/pipeline_configuration.py [hunk 4]
```diff
@@ -48,7 +50,8 @@ class PipelineConfiguration(object):
                    icr_filename="capyei_valuable.csv",
                    analysis_file_key="capyei_valuable_",
                    cleaner=None,
-                   code_scheme=None),
+                   binary_code_scheme=CodeSchemes.VALUEABLE,
```
Comment (as2388) [inline src/lib/pipeline_configuration.py]: Don't you also need to set the binary_coded_field for the import to work?

**Review by as2388 — APPROVED** [review]

File: run_scripts/test_contact_rapid_pro_ids.json (hunk 2 above)
Comment (ritazagoni) [reply run_scripts/test_contact_rapid_pro_ids.json]: Indeed!

File: src/lib/pipeline_configuration.py (hunk 3 above)
Comment (ritazagoni) [reply src/lib/pipeline_configuration.py]: Nice catch, thanks!

File: src/lib/pipeline_configuration.py (hunk 4 above)
Comment (ritazagoni) [reply src/lib/pipeline_configuration.py]: Turns out we won't be using multiple/binary schemes, removing.

**Pull request merged**